services/src/test/java/org/apache/druid/testing/embedded/EmbeddedDruidCluster.java

```diff
@@ -86,6 +86,7 @@ private EmbeddedDruidCluster()
   {
     resources.add(testFolder);
     clusterApis = new EmbeddedClusterApis(this);
+    addExtension(RuntimeInfoModule.class);
```

Comment:
Please add a list item for this in the class-level javadoc.

Reply:
Added this

```
 * <li>{@link RuntimeInfoModule} supplying a {@link RuntimeInfo} with values matching
 * {@link EmbeddedDruidServer#setServerMemory} and {@link EmbeddedDruidServer#setServerDirectMemory}</li>
```